Makefile

```diff
@@ -310,13 +310,15 @@ benchmark:
 
 lint:
 	golangci-lint run --out-format=tab --issues-exit-code=0
-	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -name '*.pb.go' | xargs gofmt -d -s
-.PHONY: lint
+
+lint-fix:
```

Comment:
This should be incorporated in the `format` rule.

Reply:
Actually, `golangci-lint run --fix --out-format=tab --issues-exit-code=0` fixes nothing for me. How many and what linters can actually automatically fix warnings>

Reply:
If and only if `golangci-lint run --fix --out-format=tab --issues-exit-code=0` really auto-fixed warnings (I doubt it, last time I checked only a few linters supported autofixing), it'd be nice to add such a line to `contrib/githook/pre-commit`. But can you check which linters support `--fix` please?

Reply:
Checking which linters support --fix  is not relevant here. Sorry, have tonnes of other tasks. 

It's good to have a linter job with `--fix` because we will not need to copy paste the command manually and adding `--fix` .